scripts/convert-pfx-cert-format.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -ex
+
+usage()
+{
+    echo "Usage:  $0 -p <pfx_cert> -k <signing_key> -c <signing_cert> -a <ca_cert>"
+    exit 1
+}
+
+while getopts "p:k:c:a:" opt; do
+    case $opt in
+        p)
+            PFX_FILE=$OPTARG
+            ;;
+        k)
+            SIGNING_KEY=$OPTARG
+            ;;
+        c)
+            SIGNING_CERT=$OPTARG
+            ;;
+        a)
+            CA_CERT=$OPTARG
+            ;;
+        *)
+            usage
+            ;;
+    esac
+done
+
+
+[ -z $PFX_FILE ] || [ -z $SIGNING_KEY ] || [ -z $SIGNING_CERT ] || [ -z $CA_CERT ] && exit 1
```

Comment:
Do you mean `( [ -z $PFX_FILE ] || [ -z $SIGNING_KEY ] || [ -z $SIGNING_CERT ] || [ -z $CA_CERT ] ) && exit 1` ?

Reply:
Yes. Fixed.